float32 lin_act_pos
float32 lin_act_vel
float32[6] joint_pos
float32[6] joint_vel